.gitignore

```diff
@@ -1,6 +1,16 @@
-notebooks/.ipynb_checkpoints/
+examples/.ipynb_checkpoints/
+examples/Input
 examples/Notebooks/.ipynb_checkpoints/
 examples/Notebooks/EyeRoundOverChuck_5.0MHz_20V.mha
+examples/0.3.2
```

Comment:
This path does not seem too intuitive.

Reply:
I mean, the version number changes frequently.

Reply:
Ah, good point. I'll remove this one.